### PR TITLE
Show detailed logs when meeting compiler errors

### PR DESCRIPTION
**Pull request by lolimay**
**Before:**
<kbd>before</kbd>
**After:**
<kbd>after</kbd>

### CODE REVIEW

File: src/server/AppManager.ts [hunk 1]
```diff
@@ -170,7 +170,9 @@ export class AppManager {
                 aff.setCompilerErrors(result.compilerErrors);
 
                 if (result.compilerErrors.length > 0) {
-                    throw new Error(`Failed to compile due to ${ result.compilerErrors.length } errors.`);
+                    const errors = result.compilerErrors.map((error) => JSON.stringify(error, null, 4)).join('\n');
```
Comment (d-gubert) [inline src/server/AppManager.ts]: Stringifying the whole error object seems to be very verbose.

Maybe just map all the `error.message`s?

**Comment by d-gubert** [comment]
Theoretically, the behavior of the method hasn't changed: it's still throwing whenever there are compiler errors. So I think we wouldn't benefit that much from adding tests for the `AppManager.load` method _in this PR_ - as it would definitely not be as trivial

**Comment by lolimay** [comment]
image

File: src/server/AppManager.ts (hunk 1 above)
Comment (lolimay) [reply src/server/AppManager.ts]: resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/166/commits/46a1e0db3e09d7d7852862823c1740098778ff0c

**Pull request merged**